**Задание 1:** Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800].
Найдите ее среднее значение и дисперсию.

In [6]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import math

In [3]:
a = 200
b = 800
M_x = (a + b) / 2
M_x

500.0

In [4]:
D_x = ((b-a)**2)/12
D_x

30000.0

**Задание 2:** О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2.
Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5?
Если да, найдите ее.

Запишем уравнение: ((b-0.5)**2)/12=0.2

Приведем к виду: b**2 - b - 2.15 = 0

Решим уравнение:

In [10]:
np.roots([1, -1, -2.15])

array([ 2.04919334, -1.04919334])

Правая граница случайной величины B = 2.04919334, так как правая граница > левой, а -1.04919334 < 0.5

Тогда среднее значение:

In [15]:
M_x = (2.04919334 + 0.5) / 2
M_x

1.27459667

**Задание 3:**  Непрерывная случайная величина X распределена нормально и задана плотностью распределения
f(x) = (1 / (4 * sqrt(2*pi))) * exp(-((x+2)**2) / 32).


Найдите:

а). M(X)

б). D(X)

в). std(X) (среднее квадратичное отклонение)

Из общего вида формулы плотности вероятности для нормального распределения:

**M_x = -2**

**D_x = 16**

**std = 4**

**Задание 4:**  Рост взрослого населения города X имеет нормальное распределение.
Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см.
Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
а). больше 182 см
б). больше 190 см
в). от 166 см до 190 см
г). от 166 см до 182 см
д). от 158 см до 190 см
е). не выше 150 см или не ниже 190 см
ё). не выше 150 см или не ниже 198 см
ж). ниже 166 см.

In [6]:
std = 8

In [4]:
0.1587

0.1587

In [5]:
0.0287

0.0287

в) **P(166 < X < 190):**

In [17]:
a = (166-174)/std
a

-1.0

In [18]:
b = (190-174)/std
b

2.0

По таблице Лапласа:

In [19]:
f_a = 0.68269
f_b = 0.95450

In [20]:
F = 1/2 * (f_a + f_b)
F

0.818595

г) **P(166 < X < 182):**

In [31]:
a = (166-174)/std
a

-1.0

In [32]:
b = (182-174)/std
b

1.0

In [33]:
f_a = 0.68269
f_b = 0.68269

In [34]:
F = 1/2 * (f_a + f_b)
F

0.68269

д) **P(158 < X < 190):**

In [27]:
a = (158-174)/std
a

-2.0

In [28]:
b = (190-174)/std
b

2.0

In [29]:
f_a = 0.95450
f_b = 0.95450

In [30]:
F = 1/2 * (f_a + f_b)
F

0.9545

е) **не выше 150 см или не ниже 190 см:**

In [35]:
a = (150-174)/std
a

-3.0

In [37]:
b = (190-174)/std
b

2.0

In [39]:
z_a = 0.0014
z_b = 0.0228

In [40]:
z_a + z_b

0.0242

ё) **не выше 150 см или не ниже 198 см:**

In [42]:
a = (150-174)/std
a

-3.0

In [43]:
b = (198-174)/std
b

3.0

In [44]:
z_a = 0.0014
z_b = 0.0014

In [45]:
z_a + z_b

0.0028

ж). **ниже 166 см:**

In [46]:
a = (166-174)/std
a

-1.0

In [47]:
z_a = 0.1587

**Задание 5:** На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от
математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

In [9]:
(190-178)/math.sqrt(25)

2.4